server/seafile-session.c

```diff
@@ -421,3 +431,24 @@ schedule_create_system_default_repo (SeafileSession *session)
                                     create_system_default_repo,
                                     NULL, session);
 }
+
+int
+create_config_table (SeafileSession *session)
+{
+    char *sql;
+    int db_type = seaf_db_type(session->db);
+
+    if (db_type == SEAF_DB_TYPE_SQLITE ||
+        db_type == SEAF_DB_TYPE_MYSQL)
+        sql = "CREATE TABLE IF NOT EXISTS SeafileConf "
```

Comment:
Actually there are many "CREATE TABLE" sqls in seafile-server, you can't run the server without  "CREATE" permission